scripts/run.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -e
+
+ARGS=$*
+
+echo "Run the RWD Docker container."
+
+if [ -z "$RWD_REPO" ]; then
+    echo "Environment variable RWD_REPO not defined."
+    exit 1
+fi
+
+if [ -z "$RWD_DATA" ]; then
+    echo "Environment variable RWD_DATA not defined."
+    exit 1
+fi
+
+# Remove container with name rwd because
+# we're about to create a new container with that name.
+RUN_COMMAND="docker run \
```

Comment:
If you use the `--rm` argument to `docker run`, it'll remove the container instance after execution.